onyo/utils.py

```diff
@@ -141,6 +140,8 @@ def validate_file(file, original_file, onyo_root):
     with open(os.path.join(onyo_root, ".onyo/validation/validation.yaml"), "r") as stream:
         try:
             rules_file = ru_yaml.load(stream)
+            if rules_file is None:
```

Comment:
I think truthiness might be better here:

```
if not rules_file:
    rules_file = []

Reply:
Good point. I will update it, and the readme commit too.